bindings/src/main/java/org/commonjava/rwx/binding/internal/reflect/ReflectionUnbinder.java

```diff
@@ -443,9 +443,6 @@ else if ( binding.getFieldType().isArray() || Collection.class.isAssignableFrom(
                         throw new XmlRpcException(
                                 "Cannot render {} (type: {}) to string. It has no corresponding coercion, and isn't an @ArrayPart or a @StructPart!" );
                     }
-
-                    logger.debug( "Attempting to coerce raw value of: {} using: {}", value, coercion );
```

Comment:
I have only empirical evidence that this change is fixing `ValueType.toString` (at least for some cases).
Please take a deeper look into it when reviewing the change :)